test/e2e/framework/log_size_monitoring.go

```diff
@@ -159,16 +159,16 @@ func (d *LogsSizeData) addNewData(ip, path string, timestamp time.Time, size int
 func NewLogsVerifier(c clientset.Interface, stopChannel chan bool) *LogsSizeVerifier {
 	nodeAddresses, err := e2essh.NodeSSHHosts(c)
 	ExpectNoError(err)
-	masterAddress := GetMasterHost() + ":22"
+	instanceAddress := APIAddress() + ":22"
 
 	workChannel := make(chan WorkItem, len(nodeAddresses)+1)
 	workers := make([]*LogSizeGatherer, workersNo)
 
 	verifier := &LogsSizeVerifier{
 		client:        c,
 		stopChannel:   stopChannel,
-		data:          prepareData(masterAddress, nodeAddresses),
-		masterAddress: masterAddress,
+		data:          prepareData(instanceAddress, nodeAddresses),
+		masterAddress: instanceAddress,
```

Comment:
not sure if you want to change the key here as well, otherwise LGTM.

Reply:
instance of what?